### ISSUE TITLE
Complete A11y Swarm for Includes

### ISSUE THREAD

**Issue by andrewscfc** [bbc/simorgh#6291]
Following the completion of https://github.com/bbc/simorgh/issues/5750 we should have a 'feature complete' includes integration for our initial 'serverside render only' includes integration. We need to complete an a11y swarm to check our integration meets our accessibility requirements.

Includes come in different forms so we need to ensure each type is tested:
- Visual Journalism
  - https://www.test.bbc.com/ws/includes/include/vjneareast/243-nuclear-infographic/english/app/embed
- IDT1
  - https://www.test.bbc.com/ws/includes/indepthtoolkit/quizzes/chemsha_bongo_mtanzania/syndicated?app-clickable=true&app-image=http%3A%2F%2Fnews.files.bbci.co.uk%2Fvj%2Flive%2Fidt-images%2Fquizzes-quiz_cta%2Fstart_geh2q.png
- IDT2
  - https://www.test.bbc.com/ws/includes/idt2/cb1a5166-cfbb-4520-bdac-6159299acff6/html
